src/Drupal/Commands/core/drush.services.yml

```diff
@@ -52,7 +52,7 @@ services:
         -  { name: drush.command }
   migrate_runner.commands:
     class: Drush\Drupal\Commands\core\MigrateRunnerCommands
-    arguments: ['@date.formatter', '@keyvalue']
+    arguments: ['@date.formatter', '@keyvalue', '@?plugin.manager.migration']
```

Comment:
I forget what this `@?` syntax means. Please add a comment.

Reply:
If the service doesn't exists (e.g. the service module is not installed) will pass `null`. I would not add a comment in the YAML as this a very known syntax and we're already using it in Drush, see https://github.com/drush-ops/drush/blob/11.x/src/Drupal/Commands/field/drush.services.yml

Reply:
Ah right, sounds good